storage/storage.go

```diff
@@ -241,6 +245,26 @@ type RefreshToken struct {
 	Nonce string
 }
 
+// RefreshTokenRef is a reference object that contains metadata about refresh tokens.
+type RefreshTokenRef struct {
+	ID        string
+	CreatedAt time.Time
+	LastUsed  time.Time
```

Comment:
hmmm... if the client ID is put into this struct, then we would only have 1 refresh token per user-client pair right? that would mean that if we need to list the refresh token per user we would have to query several `OfflineSession` objects. Or do you mean we should have a list of Client IDs per object?